src/modules/node-ci.ts

```diff
@@ -0,0 +1,37 @@
+import shell from "shelljs";
+import { CommandStringBuilder } from "../utilities/command-string-builder";
+import { OptionStringBuilder } from "../utilities/option-string-builder";
+import { Echo } from "./echo";
+
+// -----------------------------------------------------------------------------------------
+// #region Functions
+// -----------------------------------------------------------------------------------------
+
+const NodeCI = {
+    cmd(): CommandStringBuilder {
+        return new CommandStringBuilder("npm", "ci");
+    },
+    description() {
+        return `Clean and restore npm dependencies (via ${this.cmd()}) in the current directory`;
+    },
+    getOptions(): OptionStringBuilder {
+        return new OptionStringBuilder("ci");
+    },
+    run() {
+        Echo.message(
+            `Restoring npm packages (via ${this.cmd()}) in ${shell.pwd()}...`
+        );
+        shell.exec(this.cmd().toString(), { silent: false });
```

Comment:
I would error check this return and exit early if it returns a non-zero status code. I realize the `node-restore` module doesn't currently do this, but it probably should. I'm thinking we may even want to use `child_process.spawnSync` to stream the output - I think `shell.exec` runs it all in one go and just dumps the output before moving on to the next thing